packages/shared/src/components/modals/EditSquadModal.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ReactElement } from 'react';
+import { Modal, ModalProps } from './common/Modal';
+import { Squad, editSquad } from '../../graphql/squads';
+import { SquadDetails } from '../squads/Details';
+import { ModalHeaderKind } from './common/types';
+import { useToastNotification } from '../../hooks/useToastNotification';
+
+export type EditSquadModalProps = {
+  squad: Squad;
+} & Pick<ModalProps, 'isOpen' | 'onRequestClose'>;
```

Comment:
Thinking maybe we should make a generic util, as we use this quite often (this specific combination)

Reply:
Done